### PR TITLE
Assignment6 Submission

### PR DESCRIPTION
**Pull request by KennethWhite**

### CODE REVIEW

File: Assignment6/Assignment6/Assignment6.csproj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <LangVersion>7.2</LangVersion>
+  </PropertyGroup>
```
Comment (KennethWhite) [inline Assignment6/Assignment6/Assignment6.csproj]: Small note that to make the struct read-only requires C# 7.2

File: Assignment6/Assignment6.Tests/ScheduleEnumTests.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            Enum.TryParse<DaysOfWeek>(input, out DaysOfWeek day);
+            Assert.IsTrue(day.HasFlag(expectedFlag));
+
+        }
+
+        [TestMethod]
+        [DataRow("Monday, Tuesday", DaysOfWeek.Monday | DaysOfWeek.Tuesday)]
+        [DataRow("Wednesday, Friday", DaysOfWeek.Wednesday | DaysOfWeek.Friday)]
+        [DataRow("Sunday, Thursday", DaysOfWeek.Sunday | DaysOfWeek.Thursday)]
+        public void ParseDayOfWeek_MultipleDays(string input, System.Enum expectedFlag)
```
Comment (breaman) [inline Assignment6/Assignment6.Tests/ScheduleEnumTests.cs]: Since this test and the test above do exactly the same thing, you could have combined them both into just 1 test.

File: Assignment6/Assignment6.Tests/ScheduleEnumTests.cs [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        {
+            Enum.TryParse<DaysOfWeek>(input, out DaysOfWeek day);
+            Assert.IsTrue(day.HasFlag(expectedFlag));
+        }
+
+        [TestMethod]
+        public void ParseDayOfWeek_AllDays()
```
Comment (breaman) [inline Assignment6/Assignment6.Tests/ScheduleEnumTests.cs]: Why not make this just another DataRow in the test above?

File: Assignment6/Assignment6.Tests/StructAssignmentTests.cs [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+        {
+            ValueChanger.ChangeClass(_Cat);
+            Assert.AreEqual(12, _Cat.Age);
+            Assert.AreEqual("Garfield", _Cat.Name);
+        }
+
+        [TestMethod]
+        public void PassStructByReference_ChangeValues_OriginalChanged()
+        {
+            Dog d = new Dog {Name = "Baxter",Age = 99 };
```
Comment (breaman) [inline Assignment6/Assignment6.Tests/StructAssignmentTests.cs]: Why is this line here in your test? Looks like d was never used, so this is just unnecessary code.
